### PR TITLE
Open source updates

### LINKED ISSUE #2: Write some unit tests

### PR DESCRIPTION
**Pull request by dsappet**
* Moving index.js into src/ directory
* Convert to apache-2.0 license
* Apply style guide airbnb with eslint and document in readme
* Add jasmine for unit testing and write some simple tests
* Updates based on linting
* Update readme to remove slack and email 
* Update readme with ci / testing / style sections as well as other misc. updates and changes.

This fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,13 +54,40 @@ The following file extensions will be operated on:
...
+
+# Is it broken?
+Please create an issue if you are experiencing any unexpected behavior.
 
 # Contributing
-TODO still, for specific questions, contact cosmos (#team-cosmos on slack).
-* Work in a branch, submit a PR to master
 
+## Where and how to make changes?
+All work shall be done in a branch and PR shall be made to master in this repository. PRs require review and approval by an owner of the project. Upon approval the code will be merged and CI will deploy.
```
Comment (cppchriscpp) [inline README.md]:
```suggestion
All work shall be done in a branch and PR shall be made to master in this repository. PRs require review and approval by an owner of the project. Upon approval the code will be merged and CI will create a new npm package version.
```

File: README.md [hunk 2]
```diff
@@ -54,13 +54,40 @@ The following file extensions will be operated on:
...
+
+## Commit messages
+Commit messages shall be descriptive and follow the format of [semantic-release][semantic-release-1] as they are used to determine version number changes on publish.
```
Comment (cppchriscpp) [inline README.md]: Not sure what's going on with this link to semantic-release?

Do we also want to give an example or two following the guidelines?

Comment (cppchriscpp) [reply README.md]: ahh nvm on the link; I had no idea you could do that

Comment (dsappet) [reply README.md]: Thanks for getting back around to reviewing this. I dunno if I want to start putting in examples of semantic release tags. I'm going back and forth on it. It's one reason I linked the docs, hoping people would go look at it. I'm slightly worried about bloating the readme. I'll open an issue about this so we don't forget about it but I'm thinking get this merged and we can expand on that area in a followup.

**Pull request merged**